src/content/blog/node-test-migration.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: "Migration to Node.js test runner: a retrospective"
+description: "A retrospective of how Astro migrated more than 500 test suites from Mocha to Node.js test runner."
+publishDate: "February 15, 2024"
+authors:
+  - ema
+  - bjorn
+lang: "en"
+---
+
+Over a month ago, we discussed a possible migration to the Node.js test runner. While we were sufficiently happy with Mocha, we are always looking to make our CI jobs faster.
+
+Relying on a test runner baked inside our runtime had some advantages for our main monorepo:
+- Two fewer dependencies to install and maintain in our monorepo: `mocha` and `chai`.
+- Maintainability, as there are way more people involved in the Node.js project (hence the Node.js test runner).
+- The test runner will improve with time, and eventually save some time in our CI workflows.
```

Comment:
When you say improve, you mean performance wise? If so I think it should be stated clearly.